src/components/modules/banners/ausMoment/AusBanner.tsx

```diff
@@ -258,6 +258,6 @@ const AusMomentBanner: React.FC<BannerRenderProps> = ({
     );
 };
 
-const validated = validatedBannerWrapper(AusMomentBanner, 'aus-moment-banner', 'contributions');
+const validated = validatedBannerWrapper(AusMomentBanner, 'aus-moment-banner');
```

Comment:
The second argument `contributions` here is what was hardcoding the banner to set the banner channel 1 timestamp